src/mds/MDLog.h

```diff
@@ -306,5 +307,8 @@ class MDLog {
 
   // -- events --
   LogEvent *cur_event = nullptr;
+
+  // The latest 10000 events' maximum size
+  uint32_t max_event_size = 0;
```

Comment:
if this size sums up the sizes of the latest 10000 events then it seems appropriate to call it `max_events_batch_size` instead

Reply:
Yeah, sounds better. Will fix it.